crates/cheatcodes/spec/src/vm.rs

```diff
@@ -1522,6 +1522,10 @@ interface Vm {
         string calldata error
     ) external pure;
 
+    /// Returns true if the left string contains the right string, otherwise returns false.
+    #[cheatcode(group = Testing, safety = Safe)]
```

Comment:
there is where we have other string cheats, like `trim`
```suggestion
    #[cheatcode(group = String, safety = Safe)]
```